src/native/libs/System.Native/pal_networking.c

```diff
@@ -1699,7 +1777,7 @@ int32_t SystemNative_Connectx(intptr_t socket, uint8_t* socketAddress, int32_t s
     }
 #endif
     // avoid possible warning about unused parameters
-    (void*)data;
+    (void)data;
```

Comment:
Did this pointer matter?

Reply:
WASI SDK has newer version of the LLVM which complains here.